internal/pkg/utils/utils.go

```diff
@@ -696,7 +699,27 @@ func ConfigureLogging(logLevel string) {
 		logrus.SetLevel(logrus.WarnLevel)
 	}
 
-	logrus.SetOutput(os.Stderr)
+	writers := []io.Writer{os.Stderr}
+	// Set the log output to write to a log file if specified.
+	if logFilePath != "" {
+		// Create the path for the log file if it does not exist
+		err := os.MkdirAll(filepath.Dir(logFilePath), 0755)
+		if err != nil {
+			logrus.WithError(err).Errorf("Failed to create path for CNI log file: %v", filepath.Dir(logFilePath))
+		}
+
+		// Create file logger with log file rotation. Defaults to rotate once files hit 100 MB.
+		fileLogger := &lumberjack.Logger{
```

Comment:
Should we be setting a MaxBackups or MaxAge in this logger so that we don't built up log files indefinitely?

Reply:
Sure, I didn't add anything since I wasn't sure what a reasonable default might be, but I'll add something and make it configurable so it can be changed if needed.